packages/manager/src/features/Profile/APITokens/utils.ts

```diff
@@ -27,6 +28,7 @@ export const basePerms = [
 
 export const basePermNameMap = {
   account: 'Account',
+  aclp: 'ACLP',
```

Comment:
this should be aclp or monitor or cloudpulse, needs some confirmation isn't it.

Reply:
As per the latest discussion, lets use this as monitor:'Monitor'.